src/plugins/intel_gpu/src/graph/impls/onednn/convolution_onednn.cpp

```diff
@@ -70,15 +73,18 @@ struct convolution_onednn : typed_primitive_onednn_impl<convolution> {
             dnnl::memory::desc desc = onednn::layout_to_memory_desc(w_zp->get_layout(), dnnl::memory::format_tag::a, true);
             args.insert({DNNL_ARG_ATTR_ZERO_POINTS | DNNL_ARG_WEIGHTS, w_zp->get_onednn_memory(desc)});
 
-            auto dnnl_mem = w_zp->get_onednn_memory(desc);
-            void *mapped_ptr = dnnl_mem.map_data();
-            if (mapped_ptr) {
-                GPU_DEBUG_TRACE_DETAIL << instance.id() << " weights_zero_points: ";
-                for (size_t i = 0; i < desc.get_size(); ++i) {
-                    GPU_DEBUG_TRACE_DETAIL << static_cast<int32_t*>(mapped_ptr)[i] << " ";
+            GPU_DEBUG_GET_INSTANCE(debug_config);
+            GPU_DEBUG_IF(debug_config->verbose >= static_cast<int>(ov::intel_gpu::LogLevel::TRACE_DETAIL)) {
+                auto dnnl_mem = w_zp->get_onednn_memory(desc);
+                void *mapped_ptr = dnnl_mem.map_data();
+                if (mapped_ptr) {
+                    GPU_DEBUG_TRACE_DETAIL << instance.id() << " weights_zero_points: ";
+                    for (size_t i = 0; i < desc.get_size(); ++i) {
+                        GPU_DEBUG_TRACE_DETAIL << static_cast<int32_t*>(mapped_ptr)[i] << " ";
+                    }
+                    GPU_DEBUG_TRACE_DETAIL << std::endl;
+                    dnnl_mem.unmap_data(mapped_ptr);
```

Comment:
maybe we can make a common function to dump zero-point of activation and weight?
--> Do we need to dump zero-point result? I think it's enough to show print it as now. And no need common function, because of only use in convolution here.

weight zero-point is not supported on onednn at this moment. so this won't be called.
--> Agree, I added the exception code and block out.